### PR TITLE
NFDIV-2635

### PR DESCRIPTION
**Pull request by velimd**
### Change description ###

Enter a description.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-2635

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/steps/applicant1/hub-page/right-column/content.ts [hunk 1]
```diff
@@ -61,7 +61,60 @@ const en = ({ isDivorce, isApplicant2, userCase, telephoneNumber, openingTimes }
...
+  },
+  certificateOfServiceDownload: {
+    reference: 'Certificate-of-Service',
+    link: '/downloads/certificate-of-service',
+    text: "View your 'certificate of service' (PDF)",
+  },
+  respondentAnswersDownload: {
+    reference: 'Respondent-Answers',
+    link: '/downloads/respondent-answers',
+    text: `Gweld yr ymateb i'r cais ${isDivorce ? 'am ysgariad' : 'application to end your civil partnership'} (PDF)`,
```
Comment (benouaer) [inline src/main/steps/applicant1/hub-page/right-column/content.ts:78]: should the last part be in welsh?

File: src/main/steps/applicant1/hub-page/right-column/content.ts [hunk 2]
```diff
@@ -61,7 +61,60 @@ const en = ({ isDivorce, isApplicant2, userCase, telephoneNumber, openingTimes }
 });
 
 // @TODO translations
```
Comment (lara-mpt) [inline src/main/steps/applicant1/hub-page/right-column/content.ts:63]:
```suggestion
```

File: src/main/steps/applicant1/hub-page/right-column/content.ts [hunk 3]
```diff
@@ -61,7 +61,60 @@ const en = ({ isDivorce, isApplicant2, userCase, telephoneNumber, openingTimes }
...
+    text: `Gweld yr ymateb i'r cais ${isDivorce ? 'am ysgariad' : 'application to end your civil partnership'} (PDF)`,
+  },
+  deemedOrDispensedDownload: {
+    reference: 'Certificate-of-Service',
+    link: `/downloads/${
+      userCase.alternativeServiceOutcomes?.[0].value.alternativeServiceType === AlternativeServiceType.DISPENSED
+        ? 'certificate-of-dispense-with-service'
+        : 'certificate-of-deemed-as-service'
+    }`,
+    text: `View the court order granting your application for
```
Comment (benouaer) [inline src/main/steps/applicant1/hub-page/right-column/content.ts:87]: should this be in welsh?

File: src/main/steps/applicant1/hub-page/right-column/content.ts [hunk 4]
```diff
@@ -61,7 +61,60 @@ const en = ({ isDivorce, isApplicant2, userCase, telephoneNumber, openingTimes }
...
+    ${
+      userCase.alternativeServiceOutcomes?.[0].value.alternativeServiceType === AlternativeServiceType.DISPENSED
+        ? 'dispensed'
+        : 'deemed'
+    } service (PDF)`,
+  },
+  certificateOfEntitlementDownload: {
+    reference: 'Certificate-of-Entitlement',
+    link: '/downloads/certificate-of-entitlement',
+    text: 'View the certificate of entitlement (PDF)',
```
Comment (benouaer) [inline src/main/steps/applicant1/hub-page/right-column/content.ts:97]: welsh?

File: src/main/steps/applicant1/hub-page/right-column/content.ts [hunk 5]
```diff
@@ -61,7 +61,60 @@ const en = ({ isDivorce, isApplicant2, userCase, telephoneNumber, openingTimes }
...
+  },
+  conditionalOrderAnswersPdf: {
+    reference: 'Conditional-order-answers',
+    link: '/downloads/conditional-order-answers',
+    text: 'View the conditional order application (PDF)',
```
Comment (benouaer) [inline src/main/steps/applicant1/hub-page/right-column/content.ts:102]: welsh?

File: src/main/steps/applicant2/enter-your-access-code/content.ts [hunk 6]
```diff
@@ -29,7 +29,31 @@ const en = () => ({
 });
 
 // @TODO translations
-const cy = en;
+const cy: typeof en = () => ({
+  title: 'Rhowch eich manylion mynediad',
+  line1: 'Your reference number and access code are in the email you received which invited you to this application.',
```
Comment (benouaer) [inline src/main/steps/applicant2/enter-your-access-code/content.ts:34]: do we not have translations for these bits? should the ticket be blocked in that case?

File: src/main/steps/applicant1/hub-page/right-column/content.ts (hunk 1 above)
Comment (lara-mpt) [inline src/main/steps/applicant1/hub-page/right-column/content.ts:78]: if you search for 'application to end your civil partnership' you can find it's translation on other pages

**Review by lara-mpt — APPROVED** [review]

File: src/main/steps/applicant1/hub-page/right-column/content.ts (hunk 4 above)
Comment (velimd) [reply src/main/steps/applicant1/hub-page/right-column/content.ts:97]: not provided

File: src/main/steps/applicant1/hub-page/right-column/content.ts (hunk 5 above)
Comment (velimd) [reply src/main/steps/applicant1/hub-page/right-column/content.ts:102]: not provided

File: src/main/steps/applicant2/enter-your-access-code/content.ts (hunk 6 above)
Comment (velimd) [reply src/main/steps/applicant2/enter-your-access-code/content.ts:34]: yh im not sure why they didnt as they went through the page and did everything else. i agree with the blocking.

Comment (velimd) [reply src/main/steps/applicant2/enter-your-access-code/content.ts:34]: note: i did leave a comment in the ticket
https://tools.hmcts.net/jira/browse/NFDIV-2635

**Review by benouaer — APPROVED** [review]

**Pull request merged**